user/reference/osx.md

```diff
@@ -39,7 +39,7 @@ os: osx
 
 ## OS X Version
 
-Travis CI uses OS X 10.12.6 (and Xcode 8.3.3) by default . You can use another version of OS X (and Xcode) by specifying the corresponding `osx_image` key from the following table:
+Travis CI uses OS X 10.13 and Xcode 9.4.1 by default . You can use another version of OS X (and Xcode) by specifying the corresponding `osx_image` key from the following table:
```

Comment:
We could populate these version numbers if we wanted to iterate through the OSX yaml and, checking for the default , but not entirely sure it is worth the hassle.